libfulltext/config.py

```diff
@@ -1,25 +1,17 @@
-"""Config module"""
+"""Config handling module"""
 
 import os
 import yaml
 
 # Default location for the configuration file
-DEFAULT_CONFIG_PATH = "~/.config/libfulltext/config.yaml"
+DEFAULT_CONFIG_PATH = os.path.expanduser("~/.config/libfulltext/config.yaml")
```

Comment:
You kind of got a point ... I did this because click in line 15 of `bin/get_fulltext.py` actually wants to return a file handle, which needs a proper path and I do not use the expansion there. So in either way we have to do it twice. I could do it in the click code in `bin/get_fulltext.py` as well, or alternatively do it only in the initialisation of the global `DEFAULT_CONFIG_PATH` variable and not in line 13 of `config.py`

Do you have a better idea?

Notice, that we might have different executables very soon, which all need the same default config location, so placing the definition of `DEFAULT_CONFIG_PATH`  into `bin/get_fulltext.py` is not really an option, I feel.

Reply:
After talking with @andrenarchy this morning, I agree to remove the second `expand_user`. It really does not make sense to have it. Thanks for the pointer @pseyfert  ;)